api/domain_handlers_test.go

```diff
@@ -98,12 +104,19 @@ func TestAddIneligibleDomain(t *testing.T) {
 	if err != nil {
 		t.Fatalf("[%s] %s", "NewRequest Failed", err)
 	}
+	// tests that a non-App Engine http request returns a 403 status code
+	invalidW := httptest.NewRecorder()
+	invalidW.Body = &bytes.Buffer{}
+	if api.RemoveIneligibleDomains(invalidW, r); invalidW.Code != 403 {
+		t.Errorf("HTTP Response INvalid: Status code for invalid http request is not 403")
```

Comment:
nit: "invalid"

Reply:
Done

Reply:
the variable name was fine - the nit was about the string that has "INvalid"

Reply:
Ah okay, now Done